balancer/http.go

```diff
@@ -0,0 +1,42 @@
+package balancer
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/R-a-dio/valkyrie/errors"
+)
+
+func (br *Balancer) getStatus() http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		relays, err := br.storage.Relay(r.Context()).All()
+		if err != nil {
+			if errors.Is(errors.NoRelays, err) {
+				http.Error(w, "no relays", http.StatusTeapot)
+				return
+			}
+			fmt.Println(err)
```

Comment:
return some http code maybe? return at all because you currently just barrel on when an error occurs